.github/Dockerfile

```diff
@@ -5,13 +5,13 @@ WORKDIR /app
 COPY . /app
 
 ARG CI_ARCH="linux/amd64"
-ENV CI_ARCH ${CI_ARCH}
+ENV CI_ARCH=${CI_ARCH}
 ARG GIT_BRANCH="test"
-ENV GIT_BRANCH ${GIT_BRANCH}
+ENV GIT_BRANCH=${GIT_BRANCH}
 ARG GIT_TAG="test"
-ENV GIT_TAG ${GIT_TAG}
+ENV GIT_TAG=${GIT_TAG}
 ARG BUILD_OPTS=""
-ENV BUILD_OPTS ${BUILD_OPTS}
+ENV BUILD_OPTS=${BUILD_OPTS}
```

Comment:
Fixing CI warnings
```
 4 warnings found (use --debug to expand):
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 8)
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 10)
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 12)
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 14)
```